terraform/tfstate.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 4,
-  "terraform_version": "1.14.1",
-  "serial": 715,
+  "terraform_version": "1.10.7",
```

Comment:
I'm guessing tofu is behind terraform? I think it'd be safer to not run the changes locally and rely on the GitHub action to enact changes.